source/module.c

```diff
@@ -841,6 +848,10 @@ const char *aws_napi_status_to_str(napi_status status) {
         case napi_bigint_expected:
```

Comment:
A few lines up, there's a `#if NAPI_VERSION >= 3`.

I got stressed out about how the new `case` here is within a conditional.

Then I did some research and realize this file [won't even compile](https://github.com/awslabs/aws-crt-nodejs/blob/96f3e3425d6d07024726ac19d8b14d453a7bb0cc/source/module.c#L46) unless NAPI_VERSION is 4 or higher.

So please delete the `#if NAPI_VERSION >= 3`, it serves no purpose anymore